src/src/EmailBundle/Command/SendRegistrationOpenCommand.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace EmailBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use AdminBundle\Entity\Event;
+use EmailBundle\Controller\EmailController;
+
+class SendRegistrationOpenCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this->setName('email:send:registration-open')
+             ->setDescription('Sends notification emails to all subscribed attendees that the registration is open.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $controller = new EmailController();
+        $controller->setContainer($this->getContainer());
+
+        $em = $this->getContainer()->get('doctrine')->getManager();
+        $events = $em->getRepository(Event::class)->getTodaysOpenEvents();
+        $attendees = $em->getRepository(Attendee:class)->getSubscribedAttendees();
+        foreach ($events as $event) {
+            $this->sendNotificationForEvent($event, $attendees, $controller);
+        }
+    }
+
+    private function sendNotificationForEvent($event, $attendees, $controller)
+    {
+        echo 'For address '.$event->getAddress()."\n";
+        echo "--------------------\n";
+
+        $registrations = $event->getRegistrations();
+        foreach ($attendees as $attendee) {
+            $context = $this->getContextForNotificationEmail($event, $attendee, $controller);
+
+            $controller->sendEmail($attendee, $context, $event->getTemplateOverride(), 'new_event');
+
+            echo 'Sent to '.$attendee->getEmail()."\n";
+        }
+
+        echo "\n";
+    }
+
+    private function getContextForNotificationEmail($event, $attendee, $controller)
+    {
+        $context = [
+            'event' => $event,
+            'event_type' => $event->getEventType(),
+        ];
+
+        $languageCode = $attendee->getLanguages()->getCode();
+        // Warning: file_get_contents(../templates/template1/languages/en_US.yaml.twig): failed to open stream: No such file or directory
```

Comment:
tu podla mna (zatial) staci, ked napiseme poznamku, ze ten command sa ma spustat z priecinka `bin` potom to bude fungovat (podla mna).

Reply:
Uz sa tu serem asi hodinu s tym ako by sa to dalo dynamicky naloadovat, ale neviem najst rozumny sposob. Nasiel som `Finder`, ktory nevie vyhladat iba priecinok, on nachadza subory, cize sa neda rozumne pouzit do `getTemplateNamesForForm`. Nasiel som `$this->get('kernel')->loadResource()`, co sice funguje podla bundle, ale taktiez vie targetovat iba file, nie folder.
Takze dame proste do guide, ze sa to ma spusat z bin, a srat na to, alebo to mam riesit?

Reply:
tiez som hladal a tiez som narazil presne na to co si napisal ty. Neries, dame do guide.

Reply:
Ok